### ISSUE TITLE
Wallet Failing while using old Massa Station Version

### ISSUE THREAD

**Issue by pivilartisant** [massalabs/station-massa-wallet#494]
**Describe the bug**
The wallet does not work when opening it through Massa Station 0.2.3 (not latest)

**To Reproduce**

The wallet is installed and there are no accounts.

1. Have and old version of Massa Station (here 0.2.3)
2. Open the homepage 
3. Open wallet 
4. See error error 500 & screen shots

**Screenshots**
<img width="1465" alt="Screenshot 2023-07-04 at 14 55 27" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/d2716073-28f5-467c-8031-ca9cb0b57549">

<img width="446" alt="Screenshot 2023-07-04 at 14 58 37" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/8b5e638a-57d3-4f91-b6c6-db690da231de">

<img width="467" alt="Screenshot 2023-07-04 at 14 59 14" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/42fa65c3-834b-4071-afe2-d2e6bd6e11d6">

**MassaStation version**
0.2.3
**Desktop (please complete the following information):**
MacOs ventura

**Comment by pivilartisant** [comment]
Currently under investigation

**Comment by pivilartisant** [comment]
As of 11 am today, I was not able to reproduce the bug. If I encounter it again I will continue my investigation. 

In parallel, I am also examining the process of checking version compatibility between Massa Station and plugins.

**Comment by Pierregvx** [comment]
```
response :
code 602
message "source in query is required"

```
isn't that related to the fix we did on the usePost @pivilartisant ?

**Comment by Pierregvx** [comment]
It seem to be that there is a problem in the install mechanism : 
when we are updating a plugin, we are checking with the store if the plugin is compatible with massastation version, 
but when we are installing a plugin from the store we are just giving the url as parameter so there is no mechanism to check if the plugin we vant to install is compatible with massastation

**Comment by pivilartisant** [comment]
Blocked : When investigating, we discovered that there seems to be no validation when installing a plugin so i think that we should create a separate issue

**Comment by Pierregvx** [comment]
> Blocked : I wasn't able to reproduce the bug. && When investigating, we discovered that there seems to be no validation when installing a plugin so i think that we should create a separate issue @Pierregvx @Thykof

yes, check out https://github.com/massalabs/station-massa-wallet/issues/504

**Comment by SlnPons** [comment]
What do we need to do to unblock this ticket? 
@pivilartisant @Pierregvx it's not clear to me + it is a real problem as no one experienced it?

**Comment by Pierregvx** [comment]
> What do we need to do to unblock this ticket? @pivilartisant @Pierregvx it's not clear to me + it is a real problem as no one experienced it?

IMO this can be closed as the fix would be in https://github.com/massalabs/station-massa-wallet/issues/504

**Comment by SlnPons** [comment]
@pivilartisant you can't close a reported bug before the solution is implemented. It's bad practice. What if the suggested solution is not correct?

**Comment by Thykof** [comment]
I closed because this will be solve by #504 (not priorized) so this one is a duplicate.
